components.json

```diff
@@ -711,6 +711,11 @@
 			"alias": "md",
 			"owner": "Golmote"
 		},
+		"markup-attributes": {
+			"title": "Markup attributes",
+			"require": "markup",
+			"owner": "RunDevelopment"
+		},
```

Comment:
If what we're doing now is designed primarily for Vue, would it be worth just making this a full-fledged Vue language for Vue SFCs? If we need this more generally for multiple languages, we can extract it then, but `markup-attributes` isn't terribly clear as to what the languages actually does.

Reply:
I think that the modification it makes to markup justifies it being a "language". It would be kinda weird if Vue added support for plain text scripts in HTML. Honestly, the whole thing should be split even further: 1 "language" with just the attributes function and 1 language that adds support for plain text scripts.

Also, it's not like the whole thing is useful for just Vue and HTML. [ASP.net could also profit from it](https://github.com/PrismJS/prism/blob/b40f8f4b471ed5cb28fe0e9b33e5c4333e3a2bf4/components/prism-aspnet.js#L43) but I'll need to make some more adjustments for this.

> but markup-attributes isn't terribly clear as to what the languages actually does.

True. I can't think of a better name thou...